src/prefect/cli/deploy.py

```diff
@@ -185,9 +185,11 @@ async def deploy(
     ci: bool = typer.Option(
         False,
         "--ci",
+        "--no-flag",
```

Comment:
```suggestion
```

Reply:
I'm not sure why we would add that?

Reply:
@serinamarie 
1. So, I'm not sure if I am getting confused, but when we run the command 'prefect deploy --ci,' we receive a deprecated warning like this:

![2](https://github.com/PrefectHQ/prefect/assets/121633121/31631894-857f-4f8c-888b-6f1be3048c56)
 the thing you wrote above is already getting printed here 

2. As for the '--no-flag,' I believe the issue mentioned using '--no-flag' instead of '--ci,' if I interpreted it correctly. That's why I added the '--no-flag.' Maybe I am wrong. Please correct me if needed.

Reply:
1. Ah, I mean what I wrote above should suffice on its own with that exact wording.

2. Running `prefect --help` will show that there is no command `--no-flag`.

<img width="649" alt="Screenshot 2023-08-02 at 11 28 53 AM" src="https://github.com/PrefectHQ/prefect/assets/42048900/aaa421a7-0873-4cd0-9208-3ef603f8a4c1">

The correct flag is `--no-prompt`.

Reply:
@serinamarie So the description of --ci option in the prefect deploy --help command need to be changed to this :
"DEPRECATED: Please use the global '--no-prompt' flag instead: 'prefect --no-prompt deploy'"

Reply:
That's right!

Reply:
@AmanSal1  Is this right  ?

![4](https://github.com/PrefectHQ/prefect/assets/121633121/28c3c70b-afb0-4b6c-b5ed-26f177781a9f)

Reply:
I might add a line break after the first sentence for more clarity but otherwise that looks good.

Reply:
@serinamarie Done with it can you review it now  ?

Reply:
Can you update the PR description with how the output of `prefect deploy --ci` will look?

Reply:
@serinamarie Ahh,  Output of prefect deploy --ci or the the description --ci option of the prefect deploy --help ??

Reply:
@serinamarie should I attach a screenshot  of it in the description I guess it would be enough ?

Reply:
Yup, in the PR description is perfect.

Reply:
@serinamarie Done with all now you can check .